span-key-extractor/src/main/resources/config/extractors.conf

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This module shouldn't have any config

Reply:
true! I got the idea, have moved it to kafka producer module